docs/source/deployment/knative.rst

```diff
@@ -81,22 +72,49 @@ Use BentoML CLI tool to get the information about IrisClassifier.
     }
 
 
-Navigate to the BentoService archive bundle location. Build and push docker image to docker hub.
+After saving the BentoService instance, you can now start a REST API server with the
+model trained and test the API server locally:
+
+.. code-block:: bash
+
+    # Start BentoML API server:
+    bentoml serve IrisClassifier:latest
 
 
 .. code-block:: bash
 
-    > cd /Users/bozhaoyu/bentoml/repository/IrisClassifier/20200121141808_FE78B
-    > docker build . -t yubozhao/iris-classifier
-    > docker push yubozhao/iris-classifier
+    # Send test request:
+    curl -i \
+      --header "Content-Type: application/json" \
+      --request POST \
+      --data '[[5.1, 3.5, 1.4, 0.2]]' \
+      http://localhost:5000/predict
+
 
+==============================
+Deploy BentoService to Knative
+==============================
 
-=================
-Deploy to Knative
-=================
+BentoML provides a convenient way of containerizing the model API server with Docker. To
+create a docker container image for the sample model above:
 
+1. Find the file directory of the SavedBundle with bentoml get command, which is directory
```

Comment:
can you use the version I rewrote in README.md?

Reply:
* Find the SavedBundle directory with bentoml get command

* Run docker build with the SavedBundle directory which contains a generated Dockerfile

* Run the generated docker image to start a docker container serving the model